### ISSUE TITLE
Add placeholder language for users and other stakeholders to wiki

### ISSUE THREAD

**Issue by carlseglem** [codeforboston/project-template#5]
In `wiki/What's-this-all-about….md`, I suggest adding some placeholder language for
## users
- Who are users?
  > could be personas, roles

For each of them
- What is the pain/need that this would address?
- What current solution/approach/alternatives are already used or available?
- What context/situations/triggers would the system be used in/from?
- Please tell a typical story (or stories) of the user in “now” or “before” situation.
- What technology, systems do the user already use? feature/smart phone, voice/text/data plan, wifi access, own computer or tablet, library computer, friendly/family/helpful person computer/tablet/phone?
- What languages do users speak? ESL?
- Any common particular accessibility needs?
## non-user stakeholders
- Who are non-user stakeholders?
  > could be individual people (Mayor, Chair of Board of some organization), personas, roles, governing/regulatory agency people, children/caretakees of users

For each of them
- What is their interest in this project?
- If relevant, how are they evaluated (by people, regulators, themselves, others)?
- How/where/when could they be reached/contacted?

**Comment by mjrouser** [comment]
You bring up a really good point @carlseglem!  These are all key considerations for a successful project.  

To my way of thinking, there are two lanes of documentation here.  One is geared at getting people quickly up to speed and ready to jam on code.  The other is geared towards the careful planning and pre-work that needs to take place before any lines of code are pushed. 

There has been some discussion of creating a project template for Hackpad.  This would work towards the latter of these lanes and, might, possibly have the benefit of opening the process to more people (ie, not github familiar people).  

Thoughts?  Would you be interested in fleshing out a  Hackpad template starting with the points you bring up in these issues?

**Comment by carlseglem** [comment]
Yeah, @mjrouser, I can see how Hackpad would be a lower hurdle to get started, and would reduce clutter of projects on Github that don’t get to the point of coding. 

When a Github project is created, though, I think it’ll be helpful to move the information from Hackpad to the newly-created Github project’s wiki (with a pointer from earlier Hackpad page/doc to the wiki page(s)). Also, keeping the master content in Github will make it easier to track changes and share the template with other CfA groups.

I think I can take a shot at creating a Hackpad template. I’ll want to talk with you and @WheresHJ tonight a little about both content and mechanics, too.

**Comment by mjrouser** [comment]
Hey @carlseglem, I think that's a great way to look at it.  Definitely run that down with Harlan.
